src/auth/index.ts

```diff
@@ -2,10 +2,11 @@ import Logger from '../utils/logger';
 import { getStrategy as mockStrategy } from './mock';
 import { getStrategy } from './azureAD';
 export const strategyForEnvironment = () => {
-    if (process.env.NODE_ENV === 'test' || (process.env.USER_ID && process.env.NODE_ENV !== 'production')) {
-        if (process.env.USER_ID) {
-            const tid = process.env.USER_ID;
-            const n = tid.length >= 46 ? 0 : 46 - tid.length;
+    if (
+        process.env.NODE_ENV === 'test' ||
+        (process.env.NO_AUTH === 'true' && process.env.NODE_ENV !== 'production')
```

Comment:
```suggestion
        (process.env.NO_AUTH && process.env.NODE_ENV !== 'production')
```
Isn't it enaugh to check for the Env to be present? Like that NO_AUTH=1 would work too...